packages/mobile/src/tokens/saga.ts

```diff
@@ -24,24 +24,24 @@ const contractWeiPerUnit: { [key in CURRENCY_ENUM]: BigNumber | null } = {
   [CURRENCY_ENUM.DOLLAR]: null,
 }
 
-function* getWeiPerUnit(token: CURRENCY_ENUM) {
+async function getWeiPerUnit(token: CURRENCY_ENUM) {
```

Comment:
I had to change these generator functions to async ones because I needed to call them from a React component, if there's a different or better way to do this that we've been using and I missed please let me know :)